.github/workflows/test-go.yaml

```diff
@@ -23,7 +23,7 @@ jobs:
       matrix:
         os: [ubuntu-latest]
         go-version: ['^1.17.8']
-        mysql: ["mysql:5.7", "mysql:8"]
+        mysql: ["mysql:5.7.21", "mysql:8"]
```

Comment:
I chose 5.7.21 because `JSON_ARRAYAGG` was added in 5.7.22. This way, if someone tries to use `JSON_ARRAYAGG` in the future, the tests will fail and block the pr. If we run it on a daily schedule, then we may miss it until someone notices a test failure on main.

Reply:
OK, how about adding it then? (instead of replacing it).

Maybe it won't make the PR's CI check take longer.
We need a long term solution/plan when we decide to add more database implementations/versions (and not make `test-go.yml` take ~hours). (Not saying we have to tackle this now.)

PS: The separate schedule we use to run tests with `-race=true` was added because we didn't want to make `test-go.yml` take ~40m on every PR.

Reply:
Actually, using MySQL 5.7.21 seems to be breaking some tests. Reverted the change to tests in ci.